test/integration/patients/worklist/worklist.js

```diff
@@ -3160,14 +3160,6 @@ context('worklist page', function() {
             first_name: 'Test',
             last_name: 'Patient',
           },
-        },
-        {
-          id: '55555',
-          type: 'clinicians',
-          attributes: {
-            role: 'employee',
-            name: 'Test Clinician',
-          },
```

Comment:
After the move to use `/test/clinicians.json` in #952, the `Test Clinician` clinician was sometimes replacing the `55555` fixture clinician that already exists [here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/fixtures/test/clinicians.json). Which would cause the find in list test to sometimes fail because additional flows were showing in the results that it wasn't expecting.